wiki/Tournaments/oPQS/2/en.md

```diff
@@ -176,14 +176,15 @@ The osu! Philippines Quickfire Showdown 2023 was ran by various community member
 - Double Time
   1. [Amatsuki - DORAEMON [Insane]](https://osu.ppy.sh/beatmapsets/853519#osu/1783814)
   2. [Sound Souler - Eternalism 2 [Gibune's Insane]](https://osu.ppy.sh/beatmapsets/1299190#osu/2694599)
+  2. [Sound Souler - Eternalism 2 [Gibune's Insane]](https://osu.ppy.sh/beatmapsets/1299190#osu/2694599)
   3. [mafumafu - Kakushigoto [NiNo's Light Insane.]](https://osu.ppy.sh/beatmapsets/770023#osu/1618984)
   4. [James Landino - Reaction feat. Slyleaf [Milan-'s Insane]](https://osu.ppy.sh/beatmapsets/972816#osu/2036668)
 - Tiebeaker
-  1. [Amatsuki - Chiisana Koi no Uta [Together]](https://osu.ppy.sh/beatmapsets/534788#osu/1169701)
+  1. *[Amatsuki - Chiisana Koi no Uta [Together]](https://osu.ppy.sh/beatmapsets/534788#osu/1169701)*
```

Comment:
is this supposed to be bold like the other tiebreakers?

Reply:
```suggestion
  1. **[Amatsuki - Chiisana Koi no Uta [Together]](https://osu.ppy.sh/beatmapsets/534788#osu/1169701)**
```